packages/web-app-files/tests/unit/components/SideBar/Shares/Collaborators/__snapshots__/ListItem.spec.js.snap

```diff
@@ -6,10 +6,10 @@ exports[`Collaborator ListItem component share inheritance indicators show when
     <avatar-image-stub userid="brian" user-name="Brian Murphy" width="48" class="files-collaborators-collaborator-indicator"></avatar-image-stub>
     <div class="oc-text-truncate oc-width-1-1">
       <div class="oc-flex oc-flex-middle"><span class="oc-text-truncate oc-m-rm"><span aria-hidden="true" class="files-collaborators-collaborator-name">Brian Murphy</span></span> <span class="oc-invisible-sr">Share receiver name: Brian Murphy (brian@owncloud.com)</span>
-        <oc-button-stub id="share-access-details-toggle-asdf" appearance="raw" class="oc-ml-xs files-collaborators-collaborator-access-details-button">
+        <oc-button-stub id="share-access-details-toggle-557aadea-f98a-4e4a-bb52-66c374f8e22d" appearance="raw" class="oc-ml-xs files-collaborators-collaborator-access-details-button">
```

Comment:
random uuids in snapshot tests will not work 😓

Reply:
on it

Reply:
👍